lib/constants.ts

```diff
@@ -9,6 +9,12 @@ export const vaultGeneratedIamSecurityTokenSizeMax = 8192;
 // Base-64
 export const vaultGeneratedIamSecurityTokenPattern = /^[A-Za-z0-9/+=]*$/;
 
+// 15 minutes in seconds for request expiry
+export const requestExpirySeconds = 900;
+export const fifteenMinutesInMilliseconds = 900_000;
+
+export const unsignedPayload = 'UNSIGNED-PAYLOAD';
```

Comment:
If it's used only in 1 place, I'd not move it to a global constant file but keep it defined in the specific file it's being used.

Otherwise I'd prefix it to know where the variable relates to, like `authChecksumUnsignedPayload`